content/ai/concepts/search-algorithms/terms/greedy-best-first-search/greedy-best-first-search.md

```diff
@@ -12,8 +12,7 @@ CatalogContent:
   - 'paths/machine-learning-ai-engineering-foundations'
 ---
 
-**Greedy best-first search** is an informed search algorithm where the evaluation function is strictly equal to the heuristic function, disregarding the edge weights in a weighted graph. To get from a start node to a target node, the lowest value resulting from some heuristic function, **h(x)**, is considered as the successive node to traverse to. The goal is to choose the quickest and shortest path to the target node.
-
+**Greedy best-first search**  is an informed search algorithm where the evaluation function is strictly equal to the heuristic function, disregarding the edge weights in a weighted graph(because we care about the heuristic value). In ordre to search for a goal node expand the node that is closest to the goal as determined by the heuristic function.Assume that is likely to lead a solution quickly.The solution from Greedy Best First Search may not be optimal since a shorter path may exist.In this algorithm search cost is minimum since solution found without expanding a node that is not on the solution path.This algorithm is minimal but not complete since it can lead to a dead end.It's called "Greedy" since at each step it tries to get as close to the goal as it can.
```

Comment:
```suggestion
**Greedy best-first search**  is an informed search algorithm where the evaluation function is strictly equal to the heuristic function, disregarding the edge weights in a weighted graph because it only cares about the heuristic value. In order to search for a goal node it expands the node that is closest to the goal as determined by the heuristic function. This assumes that it is likely to lead to a solution quickly. However, the solution from Greedy best-first search may not be optimal since a shorter path may exist. 

In this algorithm the search cost is at a minimum since the solution is found without expanding a node that is not on the solution path. This algorithm is minimal, but not complete, since it can lead to a dead end. It's called "Greedy" because at each step it tries to get as close to the goal as it can.
```